default.env

```diff
@@ -75,3 +75,11 @@ export KLOG_VERBOSITY='1'
 # Example: 5d
 # Required: no
 export IMAGE_TAG_EXPIRATION='6h'
+
+# A github appid used to set up Pac integerating with Github App
+# Required: no
+export PAC_GITHUB_APP_ID=''
+
+# # A github private key used to set up Pac integerating with Github App
+# Required: no
```

Comment:
I would rather mention that these are required for successful run of build/rhtap-demo tests.

And I think we need a good description of how to get github app id and github app private key, i.e. explain here in a detail how someone not familiar with "PaC with github app" concept can set this up for local development

Reply:
fixed.

Reply:
I think that for users that are not familiar with configuring the "PaC with github app", it'd be still hard to do this. should we have some information here about how to set up the PaC app itself? I think that at least adding a link to the [PaC docs is must have](https://pipelinesascode.com/docs/install/github_apps/)

also I just remembered that updating [this section of the readme](https://github.com/redhat-appstudio/e2e-tests/blob/main/README.md#building-and-running-the-e2e-tests) would be nice (i.e. add information about PaC app as a prerequisite for some tests. and it might be worth mentioning that [here](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/rhtap-demo/README.md#prerequisites-for-running-the-advanced-build-scenario-against-your-own-cluster) as well

Reply:
@psturc The configuration of  "PaC with github app" is automated in script [setup-pac-integration.sh](https://github.com/redhat-appstudio/infra-deployments/blob/main/hack/build/setup-pac-integration.sh) , which is helpful to simplify its configuration. We can add the link to the [PaC docs is must have](https://pipelinesascode.com/docs/install/github_apps/)  in [this section of the readme](https://github.com/redhat-appstudio/e2e-tests/blob/main/README.md#building-and-running-the-e2e-tests)

Reply:
Beside rhtap-demo tests , build test also depends on "PaC with github app", I think we just need to add the link of "PaC with github app" to [the section of the readme](https://github.com/redhat-appstudio/e2e-tests/blob/main/README.md#building-and-running-the-e2e-tests)